### DISCUSSION TITLE
[ponzi Alert] fsbcompany

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#17] [General]
پروژه fsb company
ادرس سایت : fsbcompany.com
توضیحات سایت :

![6](https://user-images.githubusercontent.com/103103225/183713835-050b2535-33d3-4041-9482-580acb666bf8.jpg)

ترجمه ماشینی:
برنامه نویسی، و به خصوص طراحی اپلیکیشن موبایل، یک کسب و کار پرسود است و بسیاری به این صنعت روی آورده اند. اما صرف ساختن و طراحی یک اپلیکیشن ممکن است درآمدی را که انتظارش را داشتید به دست نیاورد زیرا نتوانسته اید برنامه خود را راه اندازی کنید. انتشار برنامه در شرکت FSB برای برنامه نویسان یک گزینه بسیار هوشمندانه برای معرفی اپلیکیشن و کسب حداکثر درآمد است.
در شرکت FSB بعید است که اپلیکیشنی برای انتشار پذیرفته نشود. از آنجایی که شرکت FSB نیازی به بررسی برنامه شما ندارد، توسعه دهندگان می توانند سریعتر به جامعه هدف خود برسند.
از آنجایی که شرکت ما بازار بسیار زیادی از فروشندگان و مصرف کنندگان برنامه های دانلودی دارد، بنابراین فرصتی برای توسعه دهندگان وجود دارد که برنامه های خود را به بازار مصرف بزرگ معرفی کنند و از فروش آن کسب درآمد کنند.
به این ترتیب ما برنامه دانلودی شما را برای فروش در وب سایت خود قرار می دهیم و درصدی از فروش اپلیکیشن را به شما پرداخت می کنیم. ابتدا باید در وب سایت ما ثبت نام کنید. سپس درخواست خود را در قسمت بلیط برای ما ارسال کنید. سپس تیم پشتیبانی با شما تماس می گیرد و جزئیات توافق را به شما می گوید.
توضیحات یکی از اعضا :
کمپانی FSB چیست؟
کمپانی FSB ، یک کمپانی آمریکایی_هلندی است که سرور سایت آن در این دو کشور است و از طریق سایت(whoisdomain) قابل رویت است و به مدت ۴ سال است در حوزه‌های مختلف خدمات دیجیتال فعالیت می‌کند.
و اما موضوعی که قراره باهم در اینجا راجبش صحبت کنیم، جدیدترین پروژه این کمپانی است که به خاطر شروع این پروژه دفتر خود را در شهر دبی راه‌اندازی کرده. 
سرمایه گذاری(investing)
این کمپانی استارت پروژه‌ای را در حوزه املاک و مستغلات دبی در متاورس زده و از این‌رو برای جذب کاربر بیشتر و رشد و ارتقا پروژه خود ،شروع به جذب سرمایه گذار کرده و برای جذب کاربر بیشتر طرح‌های درآمدی فوق‌العاده‌ای را در اختیار کاربران خود گذاشته.
طرح‌های در آمدی کمپانی:
در ابتدا لازم به ذکر است هر فرد می‌تواند در حساب کاربری خود از حداقل ۱۰۰تتر(دلار دیجیتال) تا حداکثر 100.000تتر سرمایه گذاری کند(طی قراردادهای۶ماهه،۱۲ماهه،۱۸ماهه،۲۴ماهه که هیچ تفاوتی جز زمان باهم ندارند)
و شما میتوانید تا ۲۳ برابر سرمایه خود را از طریق طرح‌های درآمدی کمپانی کسب نمایید. 
●طرح۱(سود ثابت) :
در این طرح به هر فرد در ازای سرمایه‌گذاری ماهیانه ۱۰٪ سود ثابت تعلق می‌گیرد.💵 که پس از پایان قرار داد اصل پول را نیز می‌توانید بردارید و قابلیت فسخ قرارداد و بازگشت وجه(revok)موجود است
*نکته:در صورت فسخ قرارداد ۳۰٪کارمزد و سودهایی که تا آن لحظه دریافت کرده‌اید کسر میشود 
●طرح۲(باینری) :
در ازای معرفی افراد با کد معرف خود میتوانید به صورت تعادلی ۱۰٪ بالانس خود را پورسانت بگیرید که تسویه آن روزانه است(شرط لازم تعادل دو طرف اکانت شماست).
این طرح درآمدی قویترین طرح ریفرال در دنیاست چون در عمق و بدون محدودیت سطح قابلیت پورسانت دهی دارد. 
●طرح۳(سود از سود):
در این طرح شما میتوانید بدون شرط ۲٪ سود ماهیانه اعضای مجموعه خود تا سطح ۱۵ را به عنوان پاداش ثابت ماهیانه دریافت کنید(برای مثال از سطح۱تا۱۵ شما اگر ۱۰۰هزار دلار سرمایه گذاری شود ماهیانه ۱۰هزار دلار سود حاصل می‌کند که ۲ درصد آن نیز که۲۰۰دلار می‌شود به شما تعلق میگیرد)

بررسی های انجام شده :
طبق توضیحات یکی از اعضا 4 سال از در حال فعالیت است که در اطلاعات whois ، سایت از سال 2020 شروع به فعالیت کرده است,

![3](https://user-images.githubusercontent.com/103103225/183713884-3e86863c-35b1-46e8-92ef-94e027388c75.jpg)

طرح های سرمایه گذاری و ثبت نام و سود دهی مشروط به شبکه سازی است که اصول و پایه شرکت های پانزی است .
بررسی در اسکم ادوایز : 
ترجمه ماشینی : 
هویت صاحب وب سایت در WHOIS پنهان است
رتبه ترانکو (چقدر ترافیک) نسبتاً پایین است
مشاغل PTC ارائه شده در این سایت ممکن است پرخطر باشند

![1](https://user-images.githubusercontent.com/103103225/183713931-15241422-a9e4-4831-9543-04752f6524ba.jpg)
![2](https://user-images.githubusercontent.com/103103225/183713938-27fddb94-60aa-480b-bbdd-86c6a5788ec3.jpg)
![4](https://user-images.githubusercontent.com/103103225/183713945-6497b99f-a7fa-4bfa-97ad-c5a3dcfcc576.jpg)
همچنان اطلاعات افراد گرداننده سایت و پروژه کاملا مخفی است که این نکته خیلی مهم است .
در ایتدای سایت خبر تاسیس دفتر دبی داده شده با یکسری تصاویر ولی هیچ گونه ادرس و تلفنی مبتی بر وجود دفتر داده نشده و صرفا به چند عکس و توضیحات بسنده کرده است .
هیچ گونه ادرس، راه ارتباطی مشخص،دفترمرکزی ، افراد گرداننده پروژه در هیچ جای سایت داده نشده و کاملا مخفی است شاید هم وجود نداشته باشد.
 برای ثبت نام حتما نیازمند معرف هستید .

![7](https://user-images.githubusercontent.com/103103225/183713972-617240aa-d36f-4252-b7d0-4303e40e0044.jpg)

تصویر زیر هم گویای خیلی از مطالب است .

![8](https://user-images.githubusercontent.com/103103225/183714015-abcf276a-d309-4195-847a-c1f889f6f463.jpg)
![10](https://user-images.githubusercontent.com/103103225/183714023-9b8afcf9-bcd8-4bdf-92f6-f1e807d9b358.jpg)

برای ثبت نام و ورود به شرکت هم نیاز به پرداخت اولیه دارید که به تترباید پرداخت شود.

![9](https://user-images.githubusercontent.com/103103225/183714052-4d3ca3b3-fc49-4dfe-a23b-7bc692bb2ee0.jpg)

ادرس ولت تتر این مجموعه که خودشون جیمیل میکنند جهت واریزی مبلغ ثبت نام  TLgnM1XENt6gLKVrPmxAxssMVz2LSW8hHM

![11](https://user-images.githubusercontent.com/103103225/183714095-bfb9a3f7-a8cc-4087-95e9-d709810c581e.jpg)

ادرس دریافت کننده مبالغ ثبت نام تمامی واریزی های خود را به ادرس TDXnWW3Dj3cLEw6hvHprR2J1ZWS326r9G4  واریز میکند و از این ادرس مبالغ در ولت های متفاوتی پخش میشود.

با توجه به اطلاعات بالا این سایت دارای هشدار پانزی و کلاهبرداری است و بر اصل شبکه سازی و ورود افراد جدید فعالیت میکند و برای ثبت نام باید مبلغی وجه واریز کرد و با توجه به این که این جا بر اساس خرید جایگاه ووورد افراد جدید فعالیت میکنند و حتی محصولی موجود نیست به شدت ریسکی است . از واریز وجه و ثبت نام و دادن اطلاعات به چنین کمپانی هایی به شدت خودداری کنید همچنین فعالیت های توضیح داده شده توسط سایت خود شرکت واهی است و واقعیت ندارد.

این بررسی در صورت یافتن اطلاعات جدید به روز خواهد شد .

**Comment by Blockunity** [comment]
به ترتیب
درباره اپلیکیشن ها، خیلی از انها کپی اپلیکیشن های اصلی هستند که بعضا رایگان برای دانلود و یا تفاوت قیمت اون اپ هم فاحش هست.
بعصی از این اپ ها از نظر اعتبار و رده بندی نظرات منفی دارند و امنیت پایین دارند.
فروش اپلیکیشن که داخل خود گوگل با قیمت پایینتر هست و سایت با قیمت بالاتر میفروشد.
اپ ها همه کپی و بی اعتبار.
درباره سایت BFH (business for home) اطلاعاتی از این شرکت ثبت نشده.

داشتن ادرس و شماره تماس دلیل بر اعتبار نیست!
توجیه اینکه اطلاعات مخفی هست فقط یه ماله کشی هست چون طبق ادعا امریکایی - هلندی هستند پس اگر کار درست هست نیاز به مخفی کاری ندارد!
اینکه ادرس و توضیحات فعالیت سایت داخل خود سایت تکمیل نیست و بعد از اکانت ساختن اطلاعات نشر داده میشه شباهت فعالیت یونیک فایننس هست.
توجیه کلاهبرداری و ماله کشی.
داشتن اعتبار tls هم دلیل نیست.
یونیک فایننس ۵ سال فعالیت داشت!
گرفتن پروتوکل امنیتی هم اعتبار نیست چون هزینه اش رو پرداخت میکنی.
وقتی قصد کلاهبرداری گسترده رو داری هزینه هم براش میکنی.
توضیحات داخل بخش پرایویسی سایت فقط اطلاعا رسانی هست و هیچ دلیلی بر ایجاد اعتماد ندارد.
فعالیت این شرکت داخل ایران در پلن های سرمایه گذاری هست که با سود ثابت درحال جذب سرمایه هست.
این گونه فعالیت ها در دسته پانزی و کلاهبرداری قرار میگیرد.
در حوزه مالی شرکتی که خدمات رسان هست تمامی اطلاعات و دیتا های مربوط به کمپانی و فاندر هارو پابلیک قرار میدهد مگر قصد کلاهبرداری دارد.
شرکت های خارجی طبق قانون بین المللی مالی حق هیچگونه فعالیت مالی با ایران را ندارد.
عکس ارائه شده برای ماه فوریه داخل دبی هست و نه ماه مارس که سایت ارائه داده ، و این فاصله اپدیت خودش نشون دهنده کلاهبردار بودن سایت هست.
شرکت بین المللی که حتی اکانت LinkedIn هم ندارد؟

**Comment by Blockunity** [comment]
نظر شخصی:

بر اساس برخورد با افراد پروموت کننده این پروژه
مشخصا یکی از ارگان های دولتی و اطلاعاتی در این شرکت سهیم هستند و یا راه اندازی کردند.
همونطور که اطلاع داریم فعالیت و سرمایه گذاری ارگان تروریستی سپاه و اطلاعات در حوزه اپلیکیشن ها و پیام رسان ها این شک رو ایجاد میکنه که افراد با سرمایه گذاری در این شرکت و نصب اپ های این شرکت بد افزار هایی رو منتشر میکنند که میتواند ابزار جاسوسی برای این ارگان مثل اسنپ ، روبیکا و خیلی از اپ های دیگه باشه.
تایید اینکه سازمان جمهوری اسلامی پشت این پروژه هست یا نه فقط یک نظریه اس چون کسانی که سنگ این پروژه رو به سینه میزنند مثل فرد تهدید کننده
https://github.com/TheGuyWhoKnowsEverything
عوامل نظام دیکتاتوری هستند و با تهدید و ایجاد وحشتی خیالی به افراد محقق که دست این کلاهبردار هارا رو میکنند، قصد نشر اطلاعات غلط دارند و خودشون رو افرادی با علم و دانش معرفی میکنند.
در انتها از سرمایه گذاری خودداری کنید
و این فرد را ریپورت threat بدید.
